apiserver/params/crossmodel.go

```diff
@@ -90,6 +90,12 @@ type AddApplicationOffer struct {
 	Endpoints              map[string]string `json:"endpoints"`
 }
 
+// DestroyApplicationOffers holds parameters for the DeleteOffers call.
+type DestroyApplicationOffers struct {
+	OfferURLs []string `json:"offer-urls"`
+	Force     bool     `json:"force,omitempty"`
```

Comment:
Do you use force yet? Or is this just for future use?

Reply:
future - wanted to get the params right beforehand